ext/sqlite3/sqlite3.stub.php

```diff
@@ -32,20 +32,20 @@ public function lastErrorMsg() {}
     public function changes() {}
 
     /** @return bool */
-    public function busyTimeout(int $ms) {}
+    public function busyTimeout(int $milliseconds) {}
 
 #ifndef SQLITE_OMIT_LOAD_EXTENSION
     /** @return bool */
-    public function loadExtension(string $shared_library) {}
+    public function loadExtension(string $name) {}
 #endif
 
 #if SQLITE_VERSION_NUMBER >= 3006011
     /** @return bool */
-    public function backup(SQLite3 $destination_db, string $source_dbname = "main", string $destination_dbname = "main") {}
+    public function backup(SQLite3 $sqlite3, string $sourceDatabase = "main", string $destinationDatabase = "main") {}
```

Comment:
The naming in this method is the last open question for me...
```suggestion
    public function backup(SQLite3 $destination, string $sourceDatabase = "main", string $destinationDatabase = "main") {}
```
Maybe?

Reply:
What's the reason of this change? Did I get it right that the first parameter is the connection of both the source and the destination databases? If that's the case, picking `$destination` is a bit confusing for me, to be honest.

But besides this, everything looks good to me!

Reply:
> What's the reason of this change? Did I get it right that the first parameter is the connection of both the source and the destination databases? If that's the case, picking $destination is a bit confusing for me, to be honest.

No, the usage here is `$source->backup($destination)`.

Reply:
Oh, I forgot that this is a method, not a function! It's a very good idea then, nice catch!

Reply:
In some cases the first parameter can also be the source, but usually you would have a separate instance of SQLite3.